docs/skills/matchers/rasanlu.md

```diff
@@ -94,6 +94,8 @@ nlu:
 
 > **Note** - Rasa NLU requires an intent to have at least three training examples in the list. There must also be a minimum of two intents in your file for Rasa to train.
 
+> **Note** - The intent matching (`@match_rasanlu('greetings')` will perform a case-sentitive RegEx search on the intent returned by Rasa.
```

Comment:
Do we want this to be the default? Or perhaps an optional thing via a kwarg?

Reply:
It should be backwards compatible this way, if it's what you mean

Reply:
I would consider this a breaking change. If I have the following matchers then things will behave differently after this change.

```python
@match_rasanlu("ask_weather")
@match_rasanlu("ask_weather_foo")
@match_rasanlu("ask_weather_bat")

or 

@match_rasanlu("hello")
@match_rasanlu("hello_world")
@match_rasanlu("world")
```

Reply:
Would it be better to have something like this to enable the regex functionality?

```python
@match_rasanlu("hello", fuzzy=True)
```

Reply:
Agree, I will change it 👌